templates/home/home-fr.html

```diff
@@ -3,7 +3,7 @@
 language: fr
 altLangPage: home-en.html
 dateModified: 2021-07-07
-layout: home
+layout: layout-home
```

Comment:
@polmih I believe the layout is `home`, not `layout-home`. With your change, the page renders with no layout at all.

Reply:
Changed.Thanks!